pr-data.csv

```diff
@@ -1504,6 +1504,77 @@ https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-t
 https://github.com/apache/flink,f91bd772de866a48d65dfcb31d4ef0d1ef2c001e,flink-core,org.apache.flink.types.RowTest.testDeepToString,ID,Opened,https://github.com/apache/flink/pull/17934,
 https://github.com/apache/flink,f91bd772de866a48d65dfcb31d4ef0d1ef2c001e,flink-core,org.apache.flink.types.RowTest.testRowNamed,ID,Opened,https://github.com/apache/flink/pull/17934,
 https://github.com/apache/geronimo-batchee,0071edabb6b340cc16f8e9d0e17ff25720b4d0fe,tools/cli,org.apache.batchee.cli.MainTest.executions,OD,FixedOrder,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-cassandra,org.apache.gora.cassandra.store.TestCassandraStore.org.apache.gora.cassandra.store.TestCassandraStore,ID,,,
+https://github.com/apache/gora,6c6b06c916b969352a6757ba7025cc78055fa87a,gora-hbase,org.apache.gora.hbase.store.TestHBaseStore.assertMetadataAnalyzer,ID,,,
```

Comment:
And these two

Reply:
`~/new-tests/gora/gora-cassandra.nondex_log`
`~/new-tests/gora/gora-hbase.nondex_log`